src/Subsystems/Lights.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Lights.h"
+#include "../RobotMap.h"
+
+Lights::Lights()
+  : Subsystem("Lights")
+{
+  feature = RobotMap::lightsFeature;
+  shooter = RobotMap::lightsShooter;
+
+  SetFeatureLights(Relay::kForward);
+}
+
+void Lights::InitDefaultCommand()
+{
+  // Set the default command for a subsystem here.
+  //SetDefaultCommand(new MySpecialCommand());
```

Comment:
This is correct. The lights do not have a default command. Lights are turned on and off only explicitly. There is no command that needs to continue running.

In the future, perhaps, lights would be controlled by a command.